src/styles/NavbarStyles/index.less

```diff
@@ -1,123 +1,188 @@
-  height: auto;
-  margin-right: 4rem;
-}
+  .navbar__links {
+    display: flex;
+    width: 40%;
+    justify-content: space-evenly;
+    margin-left: 0.6rem;
 
-.socialIcon {
-  font-size: 2rem;
-  margin: 0.7rem;
-  color: rgb(58, 9, 42);
-}
+    .navbar__navLink {
+      color: #3b0e2b;
+      font-size: 0.95rem;
+      font-weight: 600;
+      transition: 0.1s all ease-in;
+    }
+    .navbar__navLink:hover {
+      color: #fff;
+      text-decoration: underline;
+      text-decoration-thickness: 10px;
+    }
+    @media screen and (max-width: 1220px) {
+      .navbar__navLink {
+        font-size: 0.85rem;
+      }
+    }
+  }
+  @media screen and (max-width: 1070px) {
+    .navbar__links {
+      width: 50%;
+      justify-content: space-between;
+    }
+  }
 
-.socialIcon:hover {
-  color: white;
-}
+  .navbar__socials {
+    display: flex;
+    width: 15%;
+    justify-content: space-evenly;
+    transition: 0.2s all ease-in;
 
-.navContact {
-  width: 70%;
-  font-size: 1rem;
-}
+    .navbar__socials__cont {
+      display: flex;
+      width: 100%;
+      justify-content: space-evenly;
+    }
 
-.navSignup {
-  width: 55%;
-  font-size: 1rem;
-}
+    .navbar__socialLink {
+      font-size: 1.7rem;
+      color: #3b0e2b;
+      transition: 0.1s all ease-in;
+    }
+    .navbar__socialLink:hover {
+      color: white;
+    }
+    @media screen and (max-width: 1100px) {
+      .navbar__socialLink {
+        margin-right: 0;
+        font-size: 1.5rem;
+      }
+    }
+  }
+  @media screen and (max-width: 1070px) {
+    .navbar__socials {
+      display: none;
+    }
+  }
 
-.navSignup__hide {
-  display: none;
-}
+  .navbar__btns {
+    display: flex;
+    width: 18%;
+    align-items: center;
+    justify-content: space-evenly;
+    margin-left: 1.5rem;
 
-.navSignup button {
-  background: #edeed7;
-  color: #006c72;
-  font-weight: bold;
-  height: 5.5vh;
-  border-radius: 5%;
-  border: none;
-  box-shadow: 0px 0px 5px gray;
-  cursor: pointer;
-}
+    .navbar__btn {
+      background: #edeed7;
+      color: #006c72;
+      font-weight: 600;
+      min-height: 38px;
+      border-radius: 7%;
+      border: none;
+      box-shadow: rgba(0, 0, 0, 0.15) 1.95px 1.95px 2.6px;
+      font-size: 0.95rem;
+      cursor: pointer;
+      transition: 0.1s all ease-in;
+      margin: 0 0.2rem;
+    }
+    .navbar__btn:hover {
+      color: #edeed7;
+      background-color: #006c72;
+    }
+    .navbar__contact {
+      width: 120px;
+    }
+    .navbar__login {
+      width: 80px;
+    }
+    .navbar__hide {
+      display: none;
+    }
+    .navbar__profile {
+      min-height: 38px;
+      aspect-ratio: 1;
+      border-radius: 50%;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      background-color: #eeedd9;
+      box-shadow: rgba(0, 0, 0, 0.15) 1.95px 1.95px 2.6px;
+      transition: 0.2s ease-in;
+    }
+    .navbar__profile:hover {
+      transform: scale(1.1);
+      cursor: pointer;
+    }
 
```

Comment:
Good job adding multiple breakpoint, for allowing for as much responsiveness & accessibility as possible